x/ccv/provider/types/proposal.go

```diff
@@ -10,13 +10,13 @@ import (
 	evidencetypes "github.com/cosmos/cosmos-sdk/x/evidence/types"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	clienttypes "github.com/cosmos/ibc-go/v4/modules/core/02-client/types"
-	ccvtypes "github.com/cosmos/interchain-security/x/ccv/types"
+	ccvtypes "github.com/cosmos/interchain-security/v2/x/ccv/types"
 )
 
 const (
-	ProposalTypeConsumerAddition = "ConsumerAddition"
-	ProposalTypeConsumerRemoval  = "ConsumerRemoval"
-	ProposalTypeEquivocation     = "Equivocation"
+	ProposalTypeConsumerAddition = "interchain_security.ccv.provider.v2.ConsumerAdditionProposal"
```

Comment:
This change was neccessary since two versions of ICS would need to register two different types of proposals for the gov module